generators/app/templates/_package.json

```diff
@@ -27,10 +27,13 @@
     "routers": false,
     "cleanTimer": 604800000,
     "logging": {
```

Comment:
The `frontendReload` params need to be added https://github.com/rooseveltframework/roosevelt/blob/master/lib/defaults/config.json#L50

Reply:
All set with 93046b3